Signal/src/environment/Migrations/OWSDatabaseMigrationRunner.m

```diff
@@ -28,14 +26,7 @@ - (instancetype)initWithStorageManager:(TSStorageManager *)storageManager
 
 - (NSArray<OWSDatabaseMigration *> *)allMigrations
 {
-    return @[
-        [[OWS100RemoveTSRecipientsMigration alloc] initWithStorageManager:self.storageManager],
-        [[OWS102MoveLoggingPreferenceToUserDefaults alloc] initWithStorageManager:self.storageManager],
-        [[OWS103EnableVideoCalling alloc] initWithStorageManager:self.storageManager],
-        // OWS104CreateRecipientIdentities is run separately. See runSafeBlockingMigrations.
-        [[OWS105AttachmentFilePaths alloc] initWithStorageManager:self.storageManager],
-        [[OWS106EnsureProfileComplete alloc] initWithStorageManager:self.storageManager]
-    ];
+    return CurrentAppContext().allMigrations;
```

Comment:
The SAE doesn't need to run legacy migrations - and they'd be tricky to share.  But going forward, as we add new migrations both `AppContexts` will need to try to run them.

Reply:
I was going to add a flag stored in `TSStorageManager`, but you're right it's simpler to just do the migration in the SAE.  Next PR.